features/gezag-minderjarige/1.3-is-naar-buitenland-geemigreerd-geweest.feature

```diff
@@ -0,0 +1,94 @@
+#language: nl
+
+Functionaliteit: 1.3 - is naar buitenland geëmigreerd geweest
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                       |
+
+  Een minderjarige die naar het buitenland geëmigreerd geweest of tijdelijk woonachtig is geweest in het buitenland kan het gezag uit het
+  buitenland hebben meegenomen. In die gevallen wordt het gezag niet vastgelegd in BRP (ook niet in het Gezagsregister). Het gezag kan dus niet
+  betrouwbaar worden achterhaald.
+
+  Vanaf 1 mei 2011 zijn de regels van het Haags Kinderbeschermingsverdrag 1996 van toepassing.
+  Vanaf deze datum wordt het toepasselijk recht bepaald door het recht van het land van de gewone verblijfplaats van het kind.
+  Daarnaast blijft het verkregen gezag behouden na wijziging van de gewone verblijfplaats. Gezag verhuist dus mee met het kind.
+  
+  Hoe achterhalen:
+  Bepaal of de persoon als geboorteland Nederland heeft en datum vestiging in Nederland voor komt.
```

Comment:
het is denk ik handig uit te leggen waarom de vraag "is naar het buitenland geëmigreerd geweest" alleen wordt toegepast op een persoon die in Nederland geboren is.

Reply:
De consequenties van de regels 1.3, 1.3a en 1.3b zijn denk ik:
1. in NL geboren, geëmigreerd geweest en daarna in NL geadopteerd --> geeft nu gezag niet te bepalen
2. in buitenland geboren, in NL geadopteerd en daarna geëmigreerd geweest --> geeft nu wel gezag op basis van de adoptie

De 1e betekent dat na emigratie gezag nooit meer bepaald kan worden, ook niet wanneer het daarna juridisch in NL is vastgesteld (via adoptie of gerechtelijke uitspraak). 
De 2e betekent dat emigratie genegeerd wordt voor een in het buitenland geboren en in Nederland geadopteerd kind.

Ik betwijfel of dit wenselijk gedrag is. Wellicht goed om deze situaties ook op te nemen in features 1.3, 1.3a of 1.3b.

Reply:
Lijkt me goed om het op te nemen in 1.3b. Daar kan geïllustreerd worden dat een gezag uitspraak gedaan kan worden op basis van adoptie, ongeacht emigratie. In vraag 1.3 en 1.3a is de notie van adoptie onbekend.

Reply:
lijkt mij prima, met alle (volgordelijke) precondities in de Given.